src/SuplaBundle/Enums/ChannelFunction.php

```diff
@@ -106,20 +106,20 @@ public function getPossibleActions(): array {
 
     public static function actions(): array {
         return [
-            self::CONTROLLINGTHEGATEWAYLOCK => [ChannelFunctionAction::OPEN()],
-            self::CONTROLLINGTHEDOORLOCK => [ChannelFunctionAction::OPEN()],
-            self::CONTROLLINGTHEGATE => [ChannelFunctionAction::OPEN(), ChannelFunctionAction::CLOSE()],
-            self::CONTROLLINGTHEGARAGEDOOR => [ChannelFunctionAction::OPEN(), ChannelFunctionAction::CLOSE()],
+            self::CONTROLLINGTHEGATEWAYLOCK => [ChannelFunctionAction::OPEN],
```

Comment:
this is likely to break channel groups panel. Generally, we prefer enums than integers as they contain more data. Schedules form should be adjusted to make use of enums, not the other way.